### ISSUE TITLE
_CloudFlareTurnstileState._getWidgetDimension

### ISSUE THREAD

**Issue by SdxCoder** [Nika0000/cloudflare_turnstile#11]
We are getting this error on our production app,

Null check operator used on a null value
```
FlutterError
0  ???                            0x0 State.setState + 1223 (framework.dart:1223)
1  ???                            0x0 _CloudFlareTurnstileState._getWidgetDimension + 409 (turnstile_widget.dart:409)
2  ???                            0x0 _CloudFlareTurnstileState.initState.<fn>.<fn> + 275 (turnstile_widget.dart:275)
```

**cloudflare_turnstile**: 2.0.1
**iOS**: 17.6.1
**Device**: iphone12
**Flutter version**: 3.24.1

**Comment by SdxCoder** [comment]
@Nika0000 i'm using 2.0.3 now but this crash is still coming. You mentioned that it's been fixed in 2.0.3 version.

**Comment by Nika0000** [comment]
> @Nika0000 i'm using 2.0.3 now but this crash is still coming. You mentioned that it's been fixed in 2.0.3 version.
> 
> And on version 3.1.1, it is just stuck on verification
> <img width="337" alt="Screenshot 2024-10-28 at 8 46 59 AM" src="https://github.com/user-attachments/assets/2396d93a-1e6f-4ec7-ab35-186a4bbca110">
> 
> 

`cloudflare_turnstile: 3.1.1` is retracted, use the [latest version](https://pub.dev/packages/cloudflare_turnstile/admin).

**Comment by Nika0000** [comment]
I am closing this issue because it was in the deprecated version. If this issue occurs again, you can open this issue again.
